packages/discovery-react-components/src/components/DocumentPreview/utils/__tests__/box.test.ts

```diff
@@ -0,0 +1,281 @@
+    location: {
+      begin: 57080,
+      end: 57739
+    }
+  },
+  {
+    left: 54,
+    right: 524.0909423828125,
+    top: 653.3270263671875,
+    bottom: 699.5999617576599,
+    page: 13,
+    className: 'text css_1062862643',
+    location: {
+      begin: 57816,
+      end: 58086
+    }
+  },
+  {
+    left: 54,
+    right: 558.0304565429688,
+    top: 89.32704162597656,
+    bottom: 109.20001363754272,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 206337,
+      end: 206646
+    }
+  },
+  {
+    left: 54,
+    right: 558.1293334960938,
+    top: 128.9270782470703,
+    bottom: 281.0399947166443,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 206723,
+      end: 208215
+    }
+  },
+  {
+    left: 54,
+    right: 558.0005493164062,
+    top: 299.5670471191406,
+    bottom: 319.43998861312866,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 208292,
+      end: 208635
+    }
+  },
+  {
+    left: 54,
+    right: 558.1326293945312,
+    top: 337.967041015625,
+    bottom: 397.67997884750366,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 208712,
+      end: 209376
+    }
+  },
+  {
+    left: 54,
+    right: 558.0505981445312,
+    top: 416.20703125,
+    bottom: 436.08000326156616,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 209453,
+      end: 209736
+    }
+  },
+  {
+    left: 54,
+    right: 558.0557250976562,
+    top: 454.6070251464844,
+    bottom: 500.8799910545349,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 209813,
+      end: 210397
+    }
+  },
+  {
+    left: 54,
+    right: 558.0355834960938,
+    top: 507.40704345703125,
+    bottom: 540.4799666404724,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 210474,
+      end: 210926
+    }
+  },
+  {
+    left: 54,
+    right: 558.0707397460938,
+    top: 559.2470703125,
+    bottom: 592.3199934959412,
+    page: 42,
+    className: 'text css_1808944156',
+    location: {
+      begin: 211003,
+      end: 211388
+    }
+  },
+  {
+    left: 54,
+    right: 558.036376953125,
+    top: 612.0469970703125,
+    bottom: 707.5200057029724,
+    page: 42,
+    className: 'text css_1062862643',
+    location: {
+      begin: 211465,
+      end: 211708
+    }
+  }
+];
+
+describe('box', () => {
+  it('find matching bbox', () => {
+    const result = [
+      {
+        left: 54,
+        right: 558.0707397460938,
+        top: 559.2470703125,
+        bottom: 592.3199934959412,
+        page: 42,
+        className: 'text css_1808944156',
+        location: {
+          begin: 211003,
+          end: 211388
+        }
+      }
+    ];
+    expect(findMatchingBbox(originalDocBbox[0] as CellPage, processedDocBbox)).toEqual(result);
+  });
+
+  it('find matching bbox where end intersection is exclusive', () => {
```

Comment:
The "exclusive end" is for `spansIntersect`, not `intersects`. So I don't think this test makes much sense.

Reply:
But, yes, we should have a test for `spansIntersect` that tests for `end` being exclusive.